secboot/encrypt_sb_test.go

```diff
@@ -54,8 +57,8 @@ func (s *encryptSuite) TestFormatEncryptedDevice(c *C) {
 				MetadataKiBSize:     2048,
 				KeyslotsAreaKiBSize: 2560,
 				KDFOptions: &sb.KDFOptions{
-					MemoryKiB: 32768, TargetDuration: 0,
-					ForceIterations: 4, Parallel: 0,
+					MemoryKiB:       32768,
```

Comment:
didn't we discuss that this also should become just 32?

Reply:
Indeed, I was wondering and it seems it's best if I open a separate PR for this PR is about the recovery key and this change is about the main key. So unless you ffeel differently I do a small extra PR with the 32mb->32kb change for the main key.

Reply:
I did this now in https://github.com/snapcore/snapd/pull/10645

Reply:
that's fine, thanks